cypress/e2e/issue-list.cy.ts

```diff
@@ -3,112 +3,166 @@ import mockIssues2 from "../fixtures/issues-page-2.json";
 import mockIssues3 from "../fixtures/issues-page-3.json";
 
 describe("Issue List", () => {
-  beforeEach(() => {
-    // setup request mocks
-    cy.intercept("GET", "https://prolog-api.profy.dev/project", {
-      fixture: "projects.json",
-    }).as("getProjects");
-    cy.intercept("GET", "https://prolog-api.profy.dev/issue?page=1", {
-      fixture: "issues-page-1.json",
-    }).as("getIssuesPage1");
-    cy.intercept("GET", "https://prolog-api.profy.dev/issue?page=2", {
-      fixture: "issues-page-2.json",
-    }).as("getIssuesPage2");
-    cy.intercept("GET", "https://prolog-api.profy.dev/issue?page=3", {
-      fixture: "issues-page-3.json",
-    }).as("getIssuesPage3");
-
-    // open issues page
-    cy.visit(`http://localhost:3000/dashboard/issues`);
-
-    // wait for request to resolve
-    cy.wait(["@getProjects", "@getIssuesPage1"]);
-
-    // set button aliases
-    cy.get("button").contains("Previous").as("prev-button");
-    cy.get("button").contains("Next").as("next-button");
-  });
+  context("with errors", () => {
+    it("renders an error message for issues", () => {
+      cy.intercept("GET", "https://prolog-api.profy.dev/issue?page=1", {
+        status: 400,
+        forceNetworkError: true,
+        retryOnNetworkFailure: false,
+      });
 
-  context("desktop resolution", () => {
-    beforeEach(() => {
-      cy.viewport(1025, 900);
+      cy.visit(`http://localhost:3000/dashboard/issues`, {
+        retryOnNetworkFailure: false,
+      });
+
+      cy.wait(5000);
```

Comment:
This is probably necessary because of the retry feature of react-query, right? So basically the app retries this request for 3 times or so and only then shows the error. Could be a interesting to turn of react-query retries when running e2e tests to speed things up. Not sure what's best practice though